src/drivers/barometer/bmp280/bmp280.cpp

```diff
@@ -109,9 +105,8 @@ class BMP280 : public cdev::CDev
 
 	uint8_t				_curr_ctrl;
 
-	struct work_s		_work;
-	unsigned			_report_ticks; // 0 - no cycling, otherwise period of sending a report
-	unsigned			_max_mesure_ticks; //ticks needed to measure
+	unsigned			_report_interval; // 0 - no cycling, otherwise period of sending a report
+	unsigned			_max_measure_interval;
```

Comment:
Did you lose the comment? What is it what does it do?

Reply:
Comment restored.